test/crypto/convert.js

```diff
@@ -131,4 +133,23 @@ describe('convert', () => {
 				.should.be.true();
 		});
 	});
+
+	describe('#bigNumberToBuffer', () => {
+		it('should convert a big number to a buffer', () => {
+			const bignumber = '58191285901858109';
+			const expectedAddressSize = 8;
```

Comment:
Since this is part of the input rather than the output, it's confusing that it's called `expected`.